puppet/modules/jenkins_job_builder/README.md

```diff
@@ -0,0 +1,34 @@
+# jenkins_job_builder
+
+`files/theforeman.org` contains job definitions for Jenkins
+
+## development setup
+
+Warning: Your local Jenkins may not work for all the jobs you try to import, because some jobs rely on plugins and global environment variables. Unfortunately, we do not have a list of plugins and variables in a source control yet.
```

Comment:
The rebuild plugin was also removed from the Jenkins index because it didn't have a release for 8 years.

Reply:
Updated.